services/config_poll.go

```diff
@@ -0,0 +1,51 @@
+package services
+
+import (
+	"github.com/orbs-network/boyarin/boyar/config"
+	"github.com/orbs-network/boyarin/supervized"
+	"github.com/orbs-network/boyarin/utils"
+	"github.com/orbs-network/scribe/log"
+	"time"
+)
+
+type ConfigurationPollService struct {
+	flags       *config.Flags
+	logger      log.Logger
+	output      chan config.NodeConfiguration
```

Comment:
This seems confusing

Reply:
could you explain? I know it's subjective, but what's the source of confusion?

Reply:
You have two fields called `output` and `Output`. I can't really understand why there are two channels with the same name.

Reply:
both fields are the same channel (at least I hope so).
One field is private (`output`) for writes. the second field is public (`Channel`) and is read-only.